src/node-file-trace.ts

```diff
@@ -5,12 +5,14 @@ import analyze, { AnalyzeResult } from './analyze';
 import resolveDependency from './resolve-dependency';
 import { isMatch } from 'micromatch';
 import { sharedLibEmit } from './utils/sharedlib-emit';
+import { join } from 'path';
 
 const { gracefulify } = require('graceful-fs');
 gracefulify(fs);
 
 function inPath (path: string, parent: string) {
-  return path.startsWith(parent) && path[parent.length] === sep;
+  const pathWithSep = join(parent, sep);
+  return path.startsWith(pathWithSep) && path !== pathWithSep;
```

Comment:
```suggestion
  return path.startsWith(parent) && path[parent.length] === sep && path.length > parent.length;
```

seems equivalent and a slightly smaller diff, but otherwise seems gd.